examples/todo-list/src/__tests__/acceptance/todo-list.acceptance.ts

```diff
@@ -90,8 +96,14 @@ describe('TodoListApplication', () => {
 
     it('updates selected todoLists', async () => {
       await todoListRepo.deleteAll();
-      await givenTodoListInstance({title: 'red-list', color: 'red'});
-      await givenTodoListInstance({title: 'green-list', color: 'green'});
+      await givenTodoListInstance(todoListRepo, {
+        title: 'red-list',
+        color: 'red',
+      });
+      await givenTodoListInstance(todoListRepo, {
+        title: 'green-list',
+        color: 'green',
+      });
```

Comment:
This looks like a formatting-only change, please revert it.

In general, we try to avoid such changes because they add noise to git history and make future code archaeology more difficult.

For example, when trying to understand which commit introduced red and green colors in the test, we can run `git blame` to find commits that changed this test. When you reformat the code, you have to run `git blame` several more times to skip over formatting changes.

Reply:
I modified [`givenTodoListInstance`](https://github.com/strongloop/loopback-next/pull/3171/files#diff-5d2ab120333a03c68af1aab82bd58ec5R120) to include the repository as the first argument, so now the addition of `todoListRepo` as part of the call is why the formatting changed.